repo-scripts/prune-dts/prune-dts.ts

```diff
@@ -426,7 +439,7 @@ function extractExportedSymbol(
   // symbol with a less restrictive type.
   const localSymbol = typeChecker.getSymbolAtLocation(typeName);
   if (localSymbol) {
```

Comment:
`if (localSymbol && localSymbol?.declarations)` should remove the need for `localSymbol!.declarations`

Reply:
Removed all `!`s and added actual checks.